src/components/ActionList/ActionList.tsx

```diff
@@ -31,10 +31,10 @@ export function ActionList({
     finalSections = sections;
   }
 
-  const {unstableGlobalTheming = false} = useFeatures();
+  const {newDesignLanguage = false} = useFeatures();
```

Comment:
Now that this is a stable name, would it be worth adding it as a required key in the features object that gets defaulted to false in the AppProvider (and PolarisTestProvider), so then all these defaults can be removed and replaced with `const {newDesignLanguage} = useFeatures();` (as it will already be set to false)

Reply:
Had a quick chat with @danrosenthal, we'll ship this as-is for now, then loop back to this later as constant rebasing is annoying :)